src/channels.js

```diff
@@ -0,0 +1,64 @@
+
+
+module.exports = function(app) {
```

Comment:
Is this for real-time thingys? Do we need it?

Reply:
We dont need it, because we are not using realtime (websocket via socket.io or primus).
But I have add it because the feathers generator put it as default.
If you want we can remove it